services/QuillLMS/spec/models/current_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Current, type: :model do
+  after { Current.reset }
```

Comment:
Why is the `Current.reset` needed here?

Reply:
Good question! I don't think it's necessary so I removed it.